language/move-prover/tests/sources/stdlib/modules/libra_account.move

```diff
@@ -569,6 +581,59 @@ module LibraAccount {
         let transaction_fee_balance = borrow_global_mut<Balance<LBR::T>>(0xFEE);
         Libra::deposit(&mut transaction_fee_balance.coin, transaction_fee);
     }
+//    spec fun epilogue { // TODO: Verify this spec. Currently, unable to verify due to the multiplication involved
+//        aborts_if txn_max_gas_units < gas_units_remaining;
+//        aborts_if txn_gas_price * (txn_max_gas_units - gas_units_remaining) > max_u64();
```

Comment:
We wouldn't have to comment out so much stuff if we had an aborts_if that was "if" instead of "iff".  This is a good example of why we want to be able to do partial specification with aborts_if.

Reply:
Yes. I think allowing partial specification with aborts_if and better locating "aborts_if" error would be very helpful.